operators/src/processing/vector_join/util.rs

```diff
@@ -0,0 +1,22 @@
+use std::collections::{HashMap, HashSet};
+
+/// Create a translation table to resolve name conflicts in the `DataCollection`
+pub(super) fn translation_table<'i>(
```

Comment:
would be a good idea to test this function standalone, especially with the edge cases of suffixeing stuff multiple times as long as collisions occur

Reply:
5477481